src/components/home-banner/home-banner.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/css/colors.scss';
+@import '/src/css/text-styles.scss';
+
+.containerTitleHome {
+  margin: 10rem 0 0 11rem;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 50%;
+}
+
+.textContainer {
+  display: flex;
+  width: 41rem;
+
+  h2 {
+    margin: 0;
+  }
+}
+
+.polygonContainer {
+  display: flex;
+  justify-content: center;
+  align-items: end;
+  position: absolute;
+  width: 82%;
+  height: 62%;
```

Comment:
Are you sure that we need these props?

Reply:
I am using them to position the polygons, I can change them for left and top.